src/main/java/com/github/manolo8/darkbot/core/objects/slotbars/Item.java

```diff
@@ -16,8 +17,7 @@
 import static com.github.manolo8.darkbot.Main.API;
 
 public class Item extends UpdatableAuto implements eu.darkbot.api.game.items.Item {
-    private static final int START = 36, END = 128 + 8;
-    private static final byte[] BUFFER = new byte[END - START];
+    private static final int START = 36, END = 136 - START;
```

Comment:
```suggestion
    private static final int START = 36, SIZE = 136 - START;
```
These are not start & end, they're start & length or size or however you want to call it